src/cpp/src/text2image/text2image_pipeline.cpp

```diff
@@ -56,7 +57,7 @@ void Text2ImagePipeline::GenerationConfig::update_generation_config(const ov::An
 }
 
 void Text2ImagePipeline::GenerationConfig::validate() const {
-    OPENVINO_ASSERT(guidance_scale >= 1.0f || negative_prompt.empty(), "Guidance scale < 1.0 ignores negative prompt");
+    OPENVINO_ASSERT(guidance_scale >= 1.0f || negative_prompt == std::nullopt, "Guidance scale < 1.0 ignores negative prompt");
```

Comment:
looks like we also need to check `negative_prompt_2` and `negative_prompt_3` for `nullopt`